web_src/css/runner.css

```diff
@@ -10,6 +10,10 @@
   margin-left: 0.5em;
 }
 
+.runner-container .runner-ops > div {
```

Comment:
`div` seems too generic of a selector. It's not obvious what the rule does when looking at it. Maybe just add a `gt-ml-3` via HTML instead?

Reply:
Done.